builtin/logical/pki/path_config_acme.go

```diff
@@ -226,34 +227,49 @@ func (b *backend) pathAcmeWrite(ctx context.Context, req *logical.Request, d *fr
 		config.EabPolicyName = eabPolicy.Name
 	}
 
+	// Validate Default Directory Behavior:
+	defaultDirectoryPolicyType, err := getDefaultDirectoryPolicyType(config.DefaultDirectoryPolicy)
+	if err != nil {
+		return nil, fmt.Errorf("invalid default_directory_policy: %w", err)
+	}
+	defaultDirectoryRoleName := ""
+	switch defaultDirectoryPolicyType {
+	case Forbid:
+	case SignVerbatim:
+	case Role:
+		defaultDirectoryRoleName, err = getDefaultDirectoryPolicyRole(config.DefaultDirectoryPolicy)
+		if err != nil {
+			return nil, fmt.Errorf("failed extracting role name from default directory policy %w", err)
+		}
+
+		_, err := getAndValidateAcmeRole(sc, defaultDirectoryRoleName)
+		if err != nil {
+			return nil, fmt.Errorf("default directory policy role %v is not a valid ACME role: %w", defaultDirectoryRoleName, err)
+		}
+
+	}
```

Comment:
nit: Perhaps add a default failure to this switch in-case in the future we miss updating this?

Reply:
Sure